tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -244,7 +245,9 @@ function kbs_k8s_deploy() {
 	popd
 	echo "::endgroup::"
 
-	[ -n "$ingress" ] && _handle_ingress "$ingress"
+	if [ "${KBS_INGRESS}" = "aks" ]; then
```

Comment:
With only 2 implementations this is probably fine, but wouldn't it be better to modify the `_handle_ingress` to skip by `$ingress` or simply to ignore missing `_handle_ingress_$ingress` functions (assume they don't need any prep work)?

Reply:
Hi @GabyCT , I like @ldoktor 's idea.

When I created that lib I wanted to avoid too many conditionals like "if ingress X then do this else ingress Y...".